src/common/servlist.c

```diff
@@ -249,6 +261,13 @@ static const struct defaultserver def[] =
 	{"Moznet", 0, 0, 0, 0, 0, TRUE},
 	{0,			"irc.mozilla.org"},
 	
+#ifdef USE_OPENSSL
+    {"Nixbits", 0, 0, 0, LOGIN_SASL, 0, TRUE},
```

Comment:
This network had ~200 users, honestly I want to avoid filling the list with too many small networks